user_docs/en/userGuide.md

```diff
@@ -567,17 +567,22 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 
 ### Speech modes {#SpeechModes}
 
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |off, beeps, talk, on-demand|
+|Default |talk|
+
+| Option |Behaviour|
+|---|---|
+|off |NVDA will not speak anything, however it will silently react to commands.|
+|beeps |NVDA will replace normal speech with short beeps.|
+|talk |NVDA will speak normally in reaction to screen changes, notifications, and actions such as moving the focus, or issuing commands.|
+|on-demand |NVDA will only speak when you use commands with a reporting function (e.g. report the title of the window); but it will not speak in reaction to actions such as moving the focus or the cursor.|

```

Comment:
can you please copy the same order as before? it matches the default order of the commands when switching